corehq/apps/domain/forms.py

```diff
@@ -1165,39 +1168,40 @@ def __init__(self, account, domain, creating_user, plan_version, current_subscri
         )
 
     def save(self, commit=True):
```

Comment:
seems like this one can just the decorator since the whole is in the `with` clause right?